multiqc/modules/rsem/rsem.py

```diff
@@ -0,0 +1,161 @@
+                pass
+            elif len(s) == 3:
+                data['Unique'] = s[0]
+                data['Multi'] = s[1]
+                data['Uncertain'] = s[2]
+            else:
+                break
+        return data
+    
+    def rsem_stats_table(self):
+        """ Take the parsed stats from the rsem report and add them to the
+        basic stats table at the top of the report """
+
+        headers = OrderedDict()
+
+        headers['Unalignable'] = {
+            'title': '{} reads unalignable '.format(config.read_count_prefix),
+            'description': 'number of unalignable reads ({})'.format(config.read_count_desc),
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+
+        headers['Alignable'] = {
+            'title': '{} reads alignable'.format(config.read_count_prefix),
+            'description': 'number of alignable reads ({})'.format(config.read_count_desc),
+            'min': 0,
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+
+        headers['Filtered'] = {
+            'title': '{} reads filtered'.format(config.read_count_prefix),
+            'description': 'number of filtered reads due to too many alignments ({})'.format(config.read_count_desc),
+            'min': 0,
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+
+        headers['Unique'] = {
+            'title': '{} reads aligned uniquely '.format(config.read_count_prefix),
+            'description': 'number of reads aligned uniquely to a gene ({})'.format(config.read_count_desc),
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+        headers['Multi'] = {
+            'title': '{} reads aligned to multiple genes'.format(config.read_count_prefix),
+            'description': 'number of reads aligned to multiple genes ({})'.format(config.read_count_desc),
+            'min': 0,
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+        headers['Uncertain'] = {
+            'title': '{} reads aligned to multiple locations'.format(config.read_count_prefix),
+            'description': 'number of reads aligned  ({}) to multiple locations in the given reference sequences, which include isoform-level multi-mapping reads'.format(config.read_count_desc),
+            'min': 0,
+            'scale': 'PuBu',
+            'modify': lambda x: float(x) * config.read_count_multiplier,
+            'shared_key': 'read_count'
+        }
+        self.general_stats_addcols(self.rsem_mapped_data, headers)
```

Comment:
This is too many columns to add to the _General Statistics_ table. As the table contains columns from lots of different tools, I try to keep each module down to one or two columns where possible.

Here I think we can plot all of these numbers, which is great! Perhaps we can calculate a `% Alignable` to have in the General Stats and just show this instead as a single reference value? Then users can scroll down to the plots to get more detail.